packages/playground/src/components/node_details_cards/iperf_details_card.vue

```diff
@@ -61,7 +61,7 @@ export default {
     const getNodeIPerfCard = async (): Promise<NodeDetailsCard[]> => {
       const res = await gridStore.grid.zos.getNodeIPerfTest({ nodeId: props.node.nodeId });
       // filter the returned result to show node other than the one being tested against
-      const array = res.result
+      let array = res.result
         .filter(
           (node: any) => node.download_speed && node.upload_speed && !node.error && node.node_id !== props.node.nodeId,
         )
```

Comment:
```suggestion
      const array = res.result
        .filter(
          (node: any) => node.download_speed && node.upload_speed && !node.error && node.node_id !== props.node.nodeId,
        ).slice(0, 4)
```

We can slice 0 ~ 4 (4 index is excluded) if array has less than 4 element it will return them only